keyboards/acheron/elongate/readme.md

```diff
@@ -0,0 +1,15 @@
+# Elongate
+
+![Elongate](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [Gondolindrim](https://github.com/yourusername)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
```

Comment:
Can we get these fields filled out please

Reply:
Done, rehauled readme